scripts/examples/ameba_example.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+#
+#    Copyright (c) 2020 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+set -x
+env
+
+app="$1"
+root=examples/$app/Ameba/
+
+if [ -z "$app" ]; then
+    echo "No mandatory app argument supplied!"
+    exit 1
+fi
+
+source ../../scripts/activate.sh
+
+cd "$AMEBA_PATH"/project/realtek_amebaD_va0_example/GCC-RELEASE/project_lp
+make clean
```

Comment:
As right now there is only one application applied in this PR, the makefile should still be capable in adapting the current build structure. 
Also, we are working on the upgrade of the build system, and it will catch up later when more applications are applied. Thanks!